scene/resources/bit_map.cpp

```diff
@@ -674,7 +674,7 @@ void BitMap::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("grow_mask", "pixels", "rect"), &BitMap::grow_mask);
 	ClassDB::bind_method(D_METHOD("convert_to_image"), &BitMap::convert_to_image);
-	ClassDB::bind_method(D_METHOD("opaque_to_polygons", "rect", "epsilon"), &BitMap::_opaque_to_polygons_bind, DEFVAL(2.0));
+	ClassDB::bind_method(D_METHOD("opaque_to_polygons", "rect", "tolerance"), &BitMap::_opaque_to_polygons_bind, DEFVAL(2.0));
```

Comment:
A hint that this could be wrong is that epsilon values should be approx 0.000001, but this defaults to 2.0.